spec/requests_confirm_healthcare_assessment_request_spec.rb

```diff
@@ -91,7 +89,7 @@
       it 'marks healthcare workflow as confirmed' do
```

Comment:
The description should probably be updated as well since we're no longer using a workflow.